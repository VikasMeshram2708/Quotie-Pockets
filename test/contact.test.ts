import { describe, it, expect } from "vitest";

describe("Contact Us API", () => {
  it("when email is already in use", async () => {
    const data = {
      name: "test",
      email: "test11@gmail.com",
      message: "test message",
    };
    const response = await fetch("http://localhost:5000/api/contact-us", {
      method: "POST",
      headers: {
        "Content-Type": "application/json",
      },
      body: JSON.stringify(data),
    });
    const result = await response.json();
    expect(result).toMatchSnapshot();
  });
});
