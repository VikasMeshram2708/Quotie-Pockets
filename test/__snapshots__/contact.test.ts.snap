// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Contact Us API > when email is already in use 1`] = `
{
  "message": "Email already in use.",
  "success": false,
}
`;
